### DISCUSSION TITLE
OTA_EXPORTER vs OTEL_*_EXPORTER

### DISCUSSION THREAD

**Discussion by alvin562** [open-telemetry/opentelemetry-java-instrumentation#6460] [Q&A]
What is the difference between the OTA_EXPORTER and OTEL_TRACES_EXPORTER/OTEL_METRICS_EXPORTER/OTEL_LOGS_EXPORTER configuration properties?  I can't find any info on  OTA_EXPORTER but it seems it does something similar to the other properties.

**Comment by mateuszrzeszutek** [comment]
Hey @alvin562 ,
We don't support a configuration setting named `OTA_EXPORTER`.

**Reply by trask** [reply]
`OTA_EXPORTER` was used long long ago... (#841)

**Reply by alvin562** [reply]
I see. So was OTA_EXPORTER an old config replaced by the OTEL configs? Just curious to know what it was used for as I see it being used in some old code. @trask

**Reply by trask** [reply]
check out the PR I linked above

**Reply by alvin562** [reply]
Ah didnt notice that. Thanks.
